File: 02SendMessage/app/src/main/java/com/ssssyy/a02sendmessage/MainActivity.java
package com.ssssyy.a02sendmessage;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.telephony.SmsManager;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    EditText phoneNum = null;
    EditText content = null;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        phoneNum = findViewById(R.id.phone);
        content = findViewById(R.id.contentText);
        Button btn = findViewById(R.id.send);
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String num = phoneNum.getText().toString();
                String con = content.getText().toString();
                SmsManager sms = SmsManager.getDefault();
                ArrayList<String> strList = sms.divideMessage(con);
                for (String s : strList) {
                    sms.sendTextMessage(num,null,s,null,null);
                }
                Toast.makeText(MainActivity.this,"发送成功！"+num +" "+ con,Toast.LENGTH_LONG).show();
            }
        });
    }
}

File: QICQ/app/src/main/java/com/ssssyy/qicq/MainActivity.kt
package com.ssssyy.qicq

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import android.widget.Toast

class MainActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val login = findViewById<Button>(R.id.login)
        login.setOnClickListener{
            val user = findViewById<TextView>(R.id.user).text.toString()
            val pwd = findViewById<TextView>(R.id.pwd).text.toString()
            if (user == "admin" && pwd == "123456"){
                val intent = Intent(this,MainActivity::class.java)
                startActivity(intent)
                finish()
            }else{
                Toast.makeText(this,"密码错误",Toast.LENGTH_SHORT).show()
            }
        }
    }
}